images/stackstorm/Dockerfile

```diff
@@ -20,7 +20,10 @@ RUN DEBIAN_FRONTEND=noninteractive \
       crudini \
       uuid-runtime \
       apache2-utils \
-      bash-completion
+      bash-completion \
+      libkrb5-dev
```

Comment:
Are you sure we should include any Ansible-required lib dependency in base Docker image?
There are a lot of other st2 integration packs which will require other libs as well during the `pip install`.

This is somewhat similar to https://github.com/StackStorm/st2-packages/issues/241

I don't have an answer, but would like to hear opinions about pros/cons.

Reply:
Eek, no, that should not have been in the commit. Thanks for pointing out. It was part of my testing.. and slipped through the cracks at commit time.

Reply:
I agree - such dependencies should not be in the base Docker image.